src/System.IO.Compression.ZipFile/tests/ZipFileInvalidFileTests.cs

```diff
@@ -219,7 +219,9 @@ public void DirectoryEntryWithData()
         [PlatformSpecific(TestPlatforms.AnyUnix)]  // Checks Unix-specific invalid file path
         public void Unix_ZipWithInvalidFileNames_ThrowsArgumentException(string zipName)
         {
-            AssertExtensions.Throws<ArgumentException>("path", () => ZipFile.ExtractToDirectory(compat(zipName) + ".zip", GetTestFilePath()));
+            string directoryPath = GetTestFilePath();
+            ZipFile.ExtractToDirectory(compat(zipName) + ".zip", directoryPath);
+            Assert.True(Directory.Exists("\0bad"));
```

Comment:
If this is no longer throwing, is the test still useful?  And if it is, it should be renamed... the code no longer "ThrowsArgumentException".